plugins/kubernetes-backend/src/service/KubernetesBuilder.ts

```diff
@@ -325,4 +338,28 @@ export class KubernetesBuilder {
 
     return objectTypesToFetch;
   }
+
+  protected async makeProxyRequest(
+    req: express.Request,
+    res: express.Response,
+  ) {
+    const services = this.getProxyServices();
+    const proxyResponse: KubernetesProxyResponse =
+      await this.proxy.handleProxyRequest(services, req);
+    res.status(proxyResponse.code).json(proxyResponse.data);
```

Comment:
Are these _always_ json? I would have expected that there would be a bunch of other response types here and there.